src/utils_dpdk.c

```diff
@@ -697,6 +717,8 @@ static void dpdk_helper_check_pipe(dpdk_helper_ctx_t *phc) {
       .fd = phc->pipes[0], .events = POLLIN,
   };
   int data_avail = poll(&fds, 1, 0);
+  DEBUG("%s:dpdk_helper_check_pipe: poll data_avail=%d", phc->shm_name,
```

Comment:
Ideally this should have a "dpdk_common:" prefix like the other debug messages.

Reply:
I don't think so. It is consistent with traces which were already present there.